pkgs/by-name/an/android-backup-extractor/package.nix

```diff
@@ -6,13 +6,13 @@
   jre,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "android-backup-extractor";
-  version = "20210909062443-4c55371";
+  version = "unstable-2025-01-15-62310d4";
```

Comment:
```suggestion
  version = "0-unstable-2025-01-15-62310d4";
```

Reply:
I updated it.

Is this convention documented somewhere, by the way?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
Thanks!